src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -331,14 +333,27 @@ private boolean compileAdditionalSources()
         return true;
     }
 
-    private String getJniPlatform( String os ) {
+    private String getJniPlatform(Triplet target) {
+        String os = target.getOs();
+        String arch = target.getArch();
         switch (os) {
-            case Constants.OS_LINUX: return "LINUX_AMD64";
-            case Constants.OS_IOS:return "DARWIN_AARCH64";
-            case Constants.OS_DARWIN: return "DARWIN_AMD64";
-            case Constants.OS_WINDOWS: return "WINDOWS_AMD64";
-            case Constants.OS_ANDROID: return "LINUX_AARCH64";
-            default: throw new IllegalArgumentException("No support yet for " + os);
+            case Constants.OS_LINUX:
+                switch (arch) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4808459)